src/components/input/input.js

```diff
@@ -289,7 +289,11 @@ function inputTextareaDirective($mdUtil, $window, $mdAria, $timeout) {
     element.after(errorsSpacer);
 
     if (!containerCtrl.label) {
-      $mdAria.expect(element, 'aria-label', element.attr('placeholder'));
+      // It needs to wait for the next tick, in order to allow any
+      // expressions in the placeholder to be evaluated.
+      $mdUtil.nextTick(function() {
+        $mdAria.expect(element, 'aria-label', element.attr('placeholder'));
+      }, false);
```

Comment:
The next tick timeout is not necessary.

```js
    if (!containerCtrl.label) {
      $mdAria.expect(element, 'aria-label', attr.placeholder);
    }
```

Using the interpolated attribute is working as well.